### ISSUE TITLE
[Accessibility Team] Narrator/NVDA is not announcing the changed value in Resource Limit (CPU / Memory)

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#301]
[ADO LINK](https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/36837891)

prio 2
sev 3

### Actual: 
Narrator/NVDA is not announcing the changed value in Resource Limit (CPU / Memory).

Note: Similar issue:

### Issue: Maps_Deployment_Show Logs_All Levels
Actual: Screen reader does not announce selection or deselection state change for list items

### Expected:

Narrator/NVDA should automatically announce the updated value, including complete context

### User Impact:

Screen reader users cannot confirm the current CPU or Memory limit they have set.

### Repro Steps:

- Start Narrator/NVDA.
- Launch the AKS desktop application and login with your v-id.
- TAB to "Home" page.
- TAB to "Projects" tab and press ENTER.
- Open any existing project. TAB to "Deploy Application" button and press ENTER.
- "Deploy Application" Screen will open.
- Tab till Configuration button and press enter.
- Tab till Resource limit step and change any of the CPU limit.
- Observe that Narrator/NVDA is not announcing the changed value in Resource Limit (CPU / Memory)
